src/common/cluster/cluster.ts

```diff
@@ -33,6 +33,8 @@ export interface ClusterDependencies {
   createKubectl: (clusterVersion: string) => Kubectl;
   createAuthorizationReview: (config: KubeConfig) => CanI;
   createListNamespaces: (config: KubeConfig) => ListNamespaces;
+  detectorRegistry: DetectorRegistry;
```

Comment:
nit: this should be marked as `readonly`

Reply:
Done